packages/nuxt-locokit/src/paths.ts

```diff
@@ -45,10 +50,9 @@ export const ROUTES_NAMES = {
 
 export const ROUTES_PATH = {
   HOME: '/',
+  WORKSPACES: '/w',
+  CREATE_WORKSPACE: '/w/create',
```

Comment:
I prefer we group related workspaces routes together, under the `WORKSPACE` keyword.

Reply:
I can move these items next to `WORKSPACE` but I am not convinced to put theme underneath because in  `WORKSPACE`, all routes are linked (those are children), they all depend on `w/:id` unlike the others WORKSPACES and CREATE_WORKSPACE.
But if you still disagree or you have a counter-argument, I will change this.

Reply:
Well, my argument is still the same, these routes are related to the workspaces, so they should be groupe under the `WORKSPACE` keyword :-)